### PR TITLE
334 product topics sidebar layout

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.live/
- After: https://334-product-topics-sidebar-layout--danaher-ls-aem--hlxsites.hlx.live/us/en/products/assay-kits/topics/how-are-assay-kits-installed

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
Looks fantastic to me!  Just checked and this is great.  The button should be purple, but let's work on that separately.

File: blocks/topic-takeaway/topic-takeaway.js
Comment (mhaack) [inline blocks/topic-takeaway/topic-takeaway.js:1]: This should not be needed, the takeway should stay in main.

File: templates/topic/topic.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  @apply mb-20 bg-danaherpurple-50 border-t-8 border-danaherpurple-500 px-6 lg:px-10 py-6 space-y-4 leading-7 text-base text-danahergray-700;
+}
+
+*:has(.product-topic-takeaway) .default-content-wrapper h2 {
+  @apply text-xl font-semibold text-danahergray-900 my-0;
+}
+
+.social-media-container h1 {
+  @apply text-gray-900 my-2 font-bold text-2xl mb-10 mt-20;
+}
```
Comment (mhaack) [inline templates/topic/topic.css:35]: Why do we need this here? Should this be generic? Only .topic related styles are go here.

File: templates/topic/topic.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { buildBlock } from '../../scripts/lib-franklin.js';
+
+const social = `
```
Comment (mhaack) [inline templates/topic/topic.js:3]: We have the same in blog template we should move that.

**Review by mhaack — CHANGES_REQUESTED** [review]

File: templates/topic/topic.js (hunk 2 above)
Comment (duynguyen) [reply templates/topic/topic.js:3]: +1, might move it to scripts.js

File: templates/topic/topic.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+.product-topic-takeaway .default-content-wrapper {
+  @apply mb-20 bg-danaherpurple-50 border-t-8 border-danaherpurple-500 px-6 lg:px-10 py-6 space-y-4 leading-7 text-base text-danahergray-700;
+}
+
+*:has(.product-topic-takeaway) .default-content-wrapper h2 {
+  @apply text-xl font-semibold text-danahergray-900 my-0;
+}
+
+.topic main p:nth-of-type(2) {
```
Comment (mhaack) [inline templates/topic/topic.css:33]: This seams to be too specific as there might be more "p" tags depending on the content.

File: templates/topic/topic.js (hunk 2 above)
Comment (mhaack) [reply templates/topic/topic.js]: Moved it to the social block where it belongs to.

**Review by duynguyen — APPROVED** [review]

File: templates/topic/topic.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  @apply mt-10;
+}
+
+.topic main p:nth-of-type(1),
```
Comment (duynguyen) [inline templates/topic/topic.css:37]: these p elements also look too hardcoded?

**Review by mhaack — APPROVED** [review]

**Pull request merged**